### DISCUSSION TITLE
Deploy manifests without chart

### DISCUSSION THREAD

**Discussion by dstockhammer** [helmfile/helmfile#937] [Questions]
Hey there, is it possible to deploy plain old Kube manifests **without** a chart? To be more specific; this would be very handy to be able to 1) add extra resources to an existing chart and b) deploy some resources that aren't directly connected to a chart.

I realize that it's possible to wrap the manifests in a local chart and deploy that chart; but not having to do that would be great :)

To keep the paradigm that every deployed resource is a helm chart _in the cluster_, I reckon that it should be possible for helmfile to generate a "virtual chart" that wraps the manifests... 🤔

**Comment by yxxhero** [comment]
@dstockhammer PR is welcome.

**Reply by yxxhero** [reply]
@dstockhammer you can try to use a dir as a chart.
